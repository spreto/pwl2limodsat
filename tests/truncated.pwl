c
c Hand generated.
c spreto 05/09/2020
c Tests for Truncated Linear Functions
c
c The first uncommented tl line is read by pwl2limodsat
c

c tl 0 1

c tl 1 1

c tl 2 1

c tl 3 1

c tl 3 3

c tl 4 4

c tl 0 1 1 1

c tl 0 1 1 2

c tl 3 5 0 5 -7 3

tl 4 9 0 1 2 3

tl 5 6 0 1 -1 2

tl 4 3 -1 1 0 1
