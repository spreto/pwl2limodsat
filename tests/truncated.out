-= Formula phi =-

Unit 1 :: Clause      :: 19 

-= MODSAT Set Phi =-

Formula 1:
Unit 1 :: Clause      :: 4 
Unit 2 :: Clause      :: 3 
Unit 3 :: Equivalence :: 1 2
Formula 2:
Unit 1 :: Clause      :: 5 
Unit 2 :: Clause      :: 4 4 
Unit 3 :: Equivalence :: 1 2
Formula 3:
Unit 1 :: Clause      :: 6 
Unit 2 :: Clause      :: 5 5 
Unit 3 :: Equivalence :: 1 2
Formula 4:
Unit 1 :: Clause      :: 7 
Unit 2 :: Clause      :: 6 6 
Unit 3 :: Equivalence :: 1 2
Formula 5:
Unit 1 :: Clause      :: 8 
Unit 2 :: Clause      :: 7 7 
Unit 3 :: Equivalence :: 1 2
Formula 6:
Unit 1 :: Clause      :: 3 
Unit 2 :: Clause      :: 4 8 
Unit 3 :: Negation    :: 2
Unit 4 :: Equivalence :: 1 3
Formula 7:
Unit 1 :: Clause      :: 10 
Unit 2 :: Clause      :: 9 
Unit 3 :: Equivalence :: 1 2
Formula 8:
Unit 1 :: Clause      :: 11 
Unit 2 :: Clause      :: 10 10 
Unit 3 :: Equivalence :: 1 2
Formula 9:
Unit 1 :: Clause      :: 12 
Unit 2 :: Clause      :: 11 11 
Unit 3 :: Equivalence :: 1 2
Formula 10:
Unit 1 :: Clause      :: 9 
Unit 2 :: Clause      :: 10 12 
Unit 3 :: Negation    :: 2
Unit 4 :: Equivalence :: 1 3
Formula 11:
Unit 1 :: Clause      :: 14 
Unit 2 :: Clause      :: 13 
Unit 3 :: Equivalence :: 1 2
Formula 12:
Unit 1 :: Clause      :: 15 
Unit 2 :: Clause      :: 14 14 
Unit 3 :: Equivalence :: 1 2
Formula 13:
Unit 1 :: Clause      :: 16 
Unit 2 :: Clause      :: 15 15 
Unit 3 :: Equivalence :: 1 2
Formula 14:
Unit 1 :: Clause      :: 15 16 
Unit 2 :: Clause      :: 2 
Unit 3 :: Equivalence :: 1 2
Formula 15:
Unit 1 :: Clause      :: 13 
Unit 2 :: Clause      :: 9 
Unit 3 :: Implication :: 1 2
Formula 16:
Unit 1 :: Clause      :: 18 
Unit 2 :: Clause      :: 6 15 
Unit 3 :: Clause      :: -17 17 
Unit 4 :: Negation    :: 3
Unit 5 :: Implication :: 2 4
Unit 6 :: Negation    :: 5
Unit 7 :: Equivalence :: 1 6
Formula 17:
Unit 1 :: Clause      :: 19 
Unit 2 :: Clause      :: 18 18 
Unit 3 :: Equivalence :: 1 2
